hello_world.cpp

```diff
@@ -0,0 +1,5 @@
+#include <stdio.h>
```

Comment:
At the beginning the file, you should explain what this program is supposed to do.

Reply:
Done